From 1c6c74e1dd82024469f48fe2cd1e333cdc0efb62 Mon Sep 17 00:00:00 2001
From: William Wu <wulf@rock-chips.com>
Date: Wed, 15 Feb 2017 21:04:22 +0800
Subject: [PATCH] phy: rockchip-inno-usb3: select USB_PHY

Some rockchip SoCs (e.g. rk322xh/rk3328) integrated with
INNO USB 3.0 PHY have a problem to detect disconnection
correctly. So we need to depend on the usb phy framework
to handle the disconnection.

Change-Id: Ie3bd015c89e1fb8d46f69fe8d274e29462bfb763
Signed-off-by: William Wu <wulf@rock-chips.com>
---
 drivers/phy/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/phy/Kconfig b/drivers/phy/Kconfig
index 1141fe0ff18f..d1c72214f6e7 100644
--- a/drivers/phy/Kconfig
+++ b/drivers/phy/Kconfig
@@ -334,6 +334,7 @@ config PHY_ROCKCHIP_INNO_USB3
 	tristate "Rockchip INNO USB 3.0 PHY Driver"
 	depends on ARCH_ROCKCHIP && OF
 	select GENERIC_PHY
+	select USB_PHY
 	help
 	  Support for Rockchip USB 3.0 PHY with Innosilicon IP block.
 
-- 
2.35.3

